yb-voyager/cmd/cutoverToTargetCmd.go

```diff
@@ -67,6 +67,6 @@ func init() {
 	registerExportDirFlag(cutoverToTargetCmd)
 	BoolVar(cutoverToTargetCmd.Flags(), &prepareForFallBack, "prepare-for-fall-back", false,
 		"prepare for fallback by streaming changes from target DB back to source DB. Not applicable for fall-forward workflow.")
-	BoolVar(cutoverToTargetCmd.Flags(), &useLogicalReplicationYBConnector, "use-logical-replication-yb-connector", false,
-		"Use YugabyteDB connector that uses logical replication. Default is gRPC connector.")
+	BoolVar(cutoverToTargetCmd.Flags(), &useYBgRPCConnector, "use-yb-grpc-connector", true,
+		"Use the gRPC connector for export from YB. Default is true. If set to false, the new logical replication based connector will be used.")
```

Comment:
Can we mention caveats of using the new logical replication based connector? 
- only supported in 2024.1.1 and higher; need to ensure that no alter table stmts that do table rewrites (like add primary key)